app-misc/clight/clight-4.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+MY_PN="${PN^}"
+S="${WORKDIR}/${P^}"
+
+DESCRIPTION="A C daemon that turns your webcam into a light sensor"
+HOMEPAGE="https://github.com/FedeDP/Clight"
+SRC_URI="https://github.com/FedeDP/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="bash-completion geoclue upower"
+
+PATCHES=( "${FILESDIR}/clight-4.0-icon-extension.patch" )
+
+DEPEND="
+	|| ( sys-auth/elogind sys-apps/systemd )
+	dev-libs/popt
+	sci-libs/gsl
+	dev-libs/libconfig
+"
+
+RDEPEND="
+	${DEPEND}
+	app-misc/clightd
+	geoclue? ( app-misc/geoclue )
+	upower? ( sys-power/upower )
+"
+
+BDEPEND="
+	${DEPEND}
+	dev-libs/libmodule
+	dev-util/cmake
+	virtual/pkgconfig
+	sys-apps/dbus
+	bash-completion? ( app-shells/bash-completion )
```

Comment:
Mostly because the project self-describes them as [(optional) build deps](https://github.com/FedeDP/Clight/wiki/Build#optional-build-deps), and it also seems to rely on it to [get the completions directory](https://github.com/FedeDP/Clight/blob/21250bb094e7d8b6d00c67366a7cc6e89cbabd1d/CMakeLists.txt#L85).

If there's a better way to go about that, let me know and I'll modify the ebuild/add a patch/send it upstream.